packages/lib/README.md

```diff
@@ -1,15 +1,21 @@
-# H5Web Component Library
+# H5Web Visualisation Components Library
```

Comment:
you can remove the word "Library" here, I think, and put a `z` in "Visualization"